amida-tech/indaba-client
# Feature/INBA-111 Add Users Upgrade

#### What's this PR do?
Use the redux store instead of local state in the AddUsers component of the project creation wizard.
Removes Grommet Boxes and adds style stub file.

#### Related JIRA tickets:
[INBA-111](https://jira.amida-tech.com/browse/INBA-111)

#### How should this be manually tested?
1. Load the wizard http://localhost:3000/create-new-project
2. Get to the AddUsers tab
3. Check that switching between tabs, showing the user group editor, and all other features still work
4. Examine code quality

#### Any background context you want to provide?
#### Screenshots (if appropriate):
